lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -455,6 +455,20 @@ describe('#validate()', () => {
       });
     });
 
+    it('returns valid authorizer when valid object provided with disabled allowUnauthenticated', () => {
```

Comment:
Let's construct tests using `runServerless` as documented here: https://github.com/serverless/serverless/blob/master/tests/README.md#unit-tests
If something is still unclear please do not hesitate to ask.

As it'll be first tests like that in this file, you can create another `describe` at bottom of test file to host them

Reply:
Thanks for sharing a new way of testing. It looks good as it tests full configuration, not just a single method outside the context.

I'm not sure if the `validate.test.js` is the right place to add tests when using `runServerless` as my understanding it imitates `serverless` framework. 

Would it be better to move these tests to `index.test.js` and have `describe` dedicated to `onUnauthenticatedRequest` and `allowUnauthenticated`?

Reply:
@kamaz yes that's a very good point. Indeed in that case `index.test.js` is better (and it's where we usually put all tests for newly introduced plugins)